rttest/src/rttest.cpp

```diff
@@ -785,6 +793,34 @@ int rttest_calculate_statistics(struct rttest_results * results)
   return thread_rttest_instance->calculate_statistics(results);
 }
 
+void Rttest::clear_statistics()
+{
+  size_t i;
+  if (this->params.iterations == 0) {
+    i = 0;
+  } else {
+    i = this->results.iteration;
+  }
+  this->cleared_iteration = i;
+
+  // Reset the properties of the current results
+  this->results.max_latency = this->sample_buffer.latency_samples[i];
+  this->results.min_latency = this->results.max_latency;
+  this->results.mean_latency = this->results.max_latency;
+  this->results.minor_pagefaults = this->sample_buffer.minor_pagefaults[i];
+  this->results.major_pagefaults = this->sample_buffer.major_pagefaults[i];
```

Comment:
I could be wrong in my understanding here, but this isn't really "clearing" the results.  What it is doing is setting a marker at the current set of samples, and then we'll only consider samples from this point forward.

If my understanding is correct, then I think we should rename this function to reflect that usage more.  Maybe "mark_statistics" or something like that?